### ISSUE TITLE
Cannot get yup working in AWS Lambda / Serverless environment

### ISSUE THREAD

**Issue by ian** [jquense/yup#275]
For some reason yup is causing issues when trying to use in a Lambda environment.  I'm running latest 0.26.0

This causes the lambda function to fail on first request:

```
Unable to import module 'function': Error
at Function.Module._resolveFilename (module.js:469:15)
at Function.Module._load (module.js:417:25)
at Module.require (module.js:497:17)
at require (internal/module.js:20:19)
at Object.<anonymous> (/var/task/node_modules/yup/lib/index.js:3:30)
at Module._compile (module.js:570:32)
at Object.Module._extensions..js (module.js:579:10)
at Module.load (module.js:487:32)
at tryModuleLoad (module.js:446:12)
at Function.Module._load (module.js:438:3)
at Module.require (module.js:497:17)
at require (internal/module.js:20:19)
at Object.<anonymous> (/var/task/function.js:242:18)
at __webpack_require__ (/var/task/function.js:20:30)
at Object.<anonymous> (/var/task/function.js:962:12)
at __webpack_require__ (/var/task/function.js:20:30)
at Object.<anonymous> (/var/task/function.js:612:16)
at __webpack_require__ (/var/task/function.js:20:30)
at Object.<anonymous> (/var/task/function.js:582:4)
at __webpack_require__ (/var/task/function.js:20:30)
at Object.<anonymous> (/var/task/function.js:436:15)
at __webpack_require__ (/var/task/function.js:20:30)
at Object.<anonymous> (/var/task/function.js:341:1)
at __webpack_require__ (/var/task/function.js:20:30)
```

The root of this is from yup/lib/index.js:

```
"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/builtin/interopRequireDefault");

exports.__esModule = true;
exports.addMethod = addMethod;
exports.lazy = exports.ref = exports.boolean = void 0;
```

Removing yup from the project fixes everything.  Let me know if you have any ideas/thoughts, thanks!

**Comment by retyui** [comment]
https://github.com/jquense/yup/issues/273

**Comment by jquense** [comment]
fixed in `0.26.1`
